doc/ref_model/chapters/chapter10.md

```diff
@@ -41,7 +41,8 @@ Gaps
 This section addresses major open issues identified in the development of the Reference Model, Reference Architecture, Reference Implementation of the Common Cloud Infrastructure Lifecycle Framework. 
 
 ## 10.3.1 Discovery [Initial language]
-The VNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. CNTT reference model specifies a model, but OpenStack does not provide APIs to support such capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
+The VNFs/CNFs and NFVI frameworks should be able to discover each other and exchange their capabilities required or offered. One of the key pain points for most of the operators is the VNF/CNF onbarding - both in terms of time and complexity. It could take weeks or months to onboard a VNF/CNF. There are lots of static and laborious checks performed to ensure the compatibility of the VNFs with the corresponding NFVI framework. 
+The onboarding of the VNFs/CNFs should be as much automated as possible. The CNTT conformant VNFs/CBFs and NFVI frameworks should be able to discover and negotiate their capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs and VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
```

Comment:
```suggestion
The onboarding of the VNFs/CNFs should be as much automated as possible. The CNTT conformant VNFs/CNFs and NFVI frameworks should be able to discover and negotiate their capabilities. Following should be supported: NFVI should be able to publish the capabilities it offers to VNFs/CNFs and VNFs/CNFs should be able to query the NFVI for specific capabilities - such as number of cores, performance parameters. Negotiation/Hand Shake API - VNFs and NFVIs should be able to negotiate on certain capabilities. For instance, VNF desires HW acceleration for high throughput, but, should be able to fall back to high throughput offered by NFVI via DPDK offering, and vice-a-versa.
```

Do we need to explain that NFVI will expose its capabilities to VIM and VIM may select proper resources to match the requirements of VNF/CNF?